sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java

```diff
@@ -274,8 +267,9 @@ public List<SclReportItem> manageBindingForLDEPF(SCL scd, EPF epf) {
         SclRootAdapter sclRootAdapter = new SclRootAdapter(scd);
         if (!epf.isSetChannels()) return sclReportItems;
         iedService.getFilteredIeds(scd, ied -> !ied.getName().contains("TEST"))
-                .forEach(tied -> ldeviceService.findLdevice(tied, LDEVICE_LDEPF)
-                        .ifPresent(tlDevice -> getExtRefWithBayReferenceInLDEPF(scd.getDataTypeTemplates(), tied, tlDevice, sclReportItems)
+                .forEach(tied -> ldeviceService.findLdevice(tied, tlDevice -> tlDevice.getInst().equals(LDEVICE_LDEPF)
```

Comment:
you should keep the String signature, as there is only one Ldevice for Ldinst; 
You can rename the variable to clarify the Ldevice
```java
.forEach(tied -> ldeviceService.findLdevice(tied, tlDevice -> tlDevice.getInst().equals(LDEVICE_LDEPF))
                        .filter(ldepfLdevice -> PrivateUtils.extractStringPrivate(ldepfLdevice.getLN0(), COMPAS_LNODE_STATUS).map(status -> !status.equals("off")).orElse(false))
                        .ifPresent(ldepfLdevice -> getExtRefWithBayReferenceInLDEPF(tied, ldepfLdevice, sclReportItems)
```